### PR TITLE
Nitpicky grammar fix for COC

### PR DESCRIPTION
**Pull request by SiriusStarr**
#1619 (accidentally) eliminated the Oxford comma in the COC, which should be re-added, for consistency with the rest of the document (as well as to eliminate potentially ambiguous wording).

### CODE REVIEW

**Review by sjakobi — APPROVED** [review]
Thank you! :)

**Comment by SiriusStarr** [comment]
Merging this since it's minor and it's been a week without objections.

**Comment by Gabriel439** [comment]
@SiriusStarr: Yeah, for future reference, you can merge immediately once you have an approval and CI passes.  You don't have to wait for changes to this repository

**Pull request merged**
